lib/kadai-core-test/src/test/java/acceptance/jobs/AbstractKadaiJobAccTest.java

```diff
@@ -69,7 +69,7 @@ class AbstractKadaiJobAccTest {
 
   @AfterEach
   void cleanupJobs() {
-    // Dirty Hack, please refactor me with ticket https://github.com/kadai-iokadai/issues/2238
+    // Dirty Hack, please refactor me with ticket https://github.com/kadai-io/kadai/issues/2238
```

Comment:
This link is dead, either we link to the TASKANA issue, or we create a new one in KADAI

Reply:
Yes, the issue has been transferred to KADAI, I will link the new issue